optimism/src/keccak/witness.rs

```diff
@@ -0,0 +1,42 @@
+use ark_ff::Field;
```

Comment:
I don't know if this is supposed to be here. It is not used and there are some missing primitives in the KeccakEnv like `update_step`.

Reply:
You're absolutely right, I don't know how this one got in here!